nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/state/provenance-event-listing/provenance-event-listing.effects.ts

```diff
@@ -205,18 +211,26 @@ export class ProvenanceEventListingEffects {
         )
     );
 
-    deleteProvenanceQuery$ = createEffect(
-        () =>
-            this.actions$.pipe(
-                ofType(ProvenanceEventListingActions.deleteProvenanceQuery),
-                concatLatestFrom(() => [this.store.select(selectProvenanceId), this.store.select(selectClusterNodeId)]),
-                tap(([, id, clusterNodeId]) => {
-                    if (id) {
-                        this.provenanceService.deleteProvenanceQuery(id, clusterNodeId).subscribe();
-                    }
-                })
-            ),
-        { dispatch: false }
+    deleteProvenanceQuery$ = createEffect(() =>
+        this.actions$.pipe(
+            ofType(ProvenanceEventListingActions.deleteProvenanceQuery),
+            concatLatestFrom(() => [
+                this.store.select(selectActiveProvenanceId),
+                this.store.select(selectClusterNodeId)
+            ]),
+            tap(([, id, clusterNodeId]) => {
+                this.dialog.closeAll();
+
+                if (id) {
+                    this.provenanceService.deleteProvenanceQuery(id, clusterNodeId).subscribe({
+                        error: (errorResponse: HttpErrorResponse) => {
+                            this.store.dispatch(ErrorActions.snackBarError({ error: errorResponse.error }));
+                        }
+                    });
+                }
```

Comment:
Handling of the error is fine. However, what are your thoughts on not showing the error to the user in this scenario? They can't do anything about it and is pretty meaningless to them as they did get their results. It might just be more confusing to show this error.